test/unit/sthDatabaseNameMapper_test.js

```diff
@@ -29,319 +29,558 @@ var sthDatabaseNameMapper = require(ROOT_PATH + '/lib/database/model/sthDatabase
-      sthConfig.DATA_MODEL = sthConfig.DATA_MODELS.COLLECTION_PER_ATTRIBUTE;
-      expect(sthDatabaseNameMapper.unmapCollectionName(
-        'mappedtestservice', sthConfig.COLLECTION_PREFIX +
-        '/mappedtestservicepath_mappedEntityId_mappedEntityType_mappedAttrName.aggr')).to.equal(
-          sthConfig.COLLECTION_PREFIX + '/testservicepath_entityId_entityType_attrName.aggr');
-    });
-
-    after(function() {
-      sthConfig.DATA_MODEL = ORIGINAL_DATA_MODEL;
-      sthConfig.NAME_MAPPING = ORIGINAL_NAME_MAPPING;
+    describe('mapping tests', function() {
+        var ORIGINAL_DATA_MODEL = sthConfig.DATA_MODEL;
+        var ORIGINAL_NAME_MAPPING = sthConfig.NAME_MAPPING;
+
+        before(function() {
+            sthConfig.NAME_MAPPING = require(ROOT_PATH + '/test/unit/nameMappings/name-mapping.json');
+        });
+
+        it("should map the 'testservice' service as 'mappedtestservice'", function() {
+            expect(sthDatabaseNameMapper.mapService('testservice')).to.equal('mappedtestservice');
+        });
+
+        it("should map the 'testservicepath' service path as '/mappedtestservicepath'", function() {
+            expect(sthDatabaseNameMapper.mapServicePath('testservice', '/testservicepath')).to.equal(
+                '/mappedtestservicepath'
+            );
+        });
+
+        it("should map the 'entityId' entity id or name as 'mappedEntityId'", function() {
+            expect(sthDatabaseNameMapper.mapEntityName('testservice', '/testservicepath', 'entityId')).to.equal(
+                'mappedEntityId'
+            );
+        });
+
+        it("should map the 'entityType' entity type as 'mappedEntityType'", function() {
+            expect(sthDatabaseNameMapper.mapEntityType('testservice', '/testservicepath', 'entityType')).to.equal(
+                'mappedEntityType'
+            );
+        });
+
+        it("should map the 'attrName' attribute name as 'mappedAttrName'", function() {
+            expect(
+                sthDatabaseNameMapper.mapAttributeName('testservice', '/testservicepath', 'entityId', 'attrName')
+            ).to.equal('mappedAttrName');
+        });
+
+        it("should map the 'attrType' attribute type as 'mappedAttrType'", function() {
+            expect(
+                sthDatabaseNameMapper.mapAttributeType('testservice', '/testservicepath', 'entityId', 'attrType')
+            ).to.equal('mappedAttrType');
+        });
+
+        it(
+            "should map the '" +
+                sthConfig.DB_PREFIX +
+                "testservice' database as '" +
+                sthConfig.DB_PREFIX +
+                "mappedtestservice'",
+            function() {
+                expect(sthDatabaseNameMapper.mapDatabaseName(sthConfig.DB_PREFIX + 'testservice')).to.equal(
+                    sthConfig.DB_PREFIX + 'mappedtestservice'
+                );
+            }
+        );
+
+        it(
+            "should map the '" +
+                sthConfig.COLLECTION_PREFIX +
```

Comment:
Excesive line break here and other similar places in this file. Please do a general review of it.

Reply:
Fixed  c00464eb